partials/nav.html

```diff
@@ -11,6 +11,7 @@ <h4>NeuroData</h4>
         {% include 'partials/nav_content.html' %}
 
         <ul class="bottom">
+            <li><a href="https://bitsandbrains.io/2018/09/02/how-to-email-faculty-for-job.html" target="_blank"><h5>Join Us</h5></a></li>
```

Comment:
this is fine for now, but we should make an issue to collapse the join us and contact to just a single line with icons (also adding the social media icons and links out to each)

Reply:
issue made